### PR TITLE
Refactored and referenced optimized implmentation.

### PR DESCRIPTION
**Pull request by amitsrivastava78**
For Relu1 and Relu removed local implmentation.

### CODE REVIEW

File: tensorflow/lite/kernels/activations.cc [hunk 1]
```diff
@@ -333,12 +333,9 @@ TfLiteStatus ReluEval(TfLiteContext* context, TfLiteNode* node) {
...
     case kTfLiteFloat32: {
-      size_t elements = input->bytes / sizeof(float);
-      float* in = input->data.f;
-      float* in_end = in + elements;
-      float* out = output->data.f;
-      for (; in < in_end; in++, out++) *out = std::max(0.f, *in);
-      return kTfLiteOk;
+      // refering to the optimized implementation
+      optimized_ops::Relu(GetTensorShape(input), GetTensorData<float>(input),
+                          GetTensorShape(output), GetTensorData<float>(output));
```
Comment (renjie-liu) [inline tensorflow/lite/kernels/activations.cc]: return kTfLiteOk?

Comment (amitsrivastava78) [reply tensorflow/lite/kernels/activations.cc]: @renjie-liu , thanks for pointing this out, i have updated the code as per the comments, Kindly check and approve.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@renjie-liu , thanks for pointing this out, i have updated the code as per the comments, Kindly check and approve.

Regards
Amit

**Review by renjie-liu — DISMISSED** [review]

**Comment by amitsrivastava78** [comment]
@renjie-liu , thanks for the review, really appreciate your efforts in making the code better.

Regards
Amit

File: tensorflow/lite/kernels/activations.cc [hunk 2]
```diff
@@ -333,11 +333,9 @@ TfLiteStatus ReluEval(TfLiteContext* context, TfLiteNode* node) {
   TfLiteTensor* output = GetOutput(context, node, 0);
   switch (input->type) {
     case kTfLiteFloat32: {
-      size_t elements = input->bytes / sizeof(float);
-      float* in = input->data.f;
-      float* in_end = in + elements;
-      float* out = output->data.f;
-      for (; in < in_end; in++, out++) *out = std::max(0.f, *in);
+      // refering to the optimized implementation
```
Comment (jdduke) [inline tensorflow/lite/kernels/activations.cc]: No need for this comment, it's clear from the code. Same below.

**Review by jdduke — CHANGES_REQUESTED** [review]

File: tensorflow/lite/kernels/activations.cc (hunk 2 above)
Comment (amitsrivastava78) [reply tensorflow/lite/kernels/activations.cc]: @jdduke , thanks for the review, i have updated the code as per your suggestion. Kindly check and approve.

Regards
Amit

**Comment by amitsrivastava78** [comment]
@jdduke , thanks for the review, i have updated the code as per your suggestion. Kindly check and approve.

Regards
Amit

**Review by jdduke — APPROVED** [review]

**Comment by amitsrivastava78** [comment]
@rthadur & @gbaned , i have checked the error, it seems it is not caused by my code change , can you pls check and re trigger the build.

_Traceback (most recent call last):
  File "check_load_py_test.py", line 91, in <module>
    main()
  File "check_load_py_test.py", line 85, in main
    '\n'.join(files_missing_load)))
RuntimeError: The following files are missing load("//tensorflow:tensorflow.bzl", "py_test").
This load statement is needed because otherwise pip tests will try to use their dependencies, which are not visible to them.:
 tensorflow/contrib/input_pipeline/BUILD
tensorflow/contrib/periodic_resample/BUILD_

**Pull request merged**